pcdsdevices/tests/conftest.py

```diff
@@ -113,14 +113,15 @@ def find_pcdsdevices_submodules() -> Dict[str, ModuleType]:
     return modules
 
 
-def find_all_classes(classes) -> list:
+def find_all_classes(classes, skip: List[str] = []) -> list:
```

Comment:
This will seem pedantic, however it's been drilled into me in the past and I think it's been for the better:
-> Never include mutable default arguments. It doesn't matter if `skip` is actually mutated here or not.  It doesn't matter if the function is simple or not.

Future refactors may forget about this and introduce bugs - so let's reinforce good habits and side-step that issue now by just not doing it at all.

(`skip: Optional[List[str]] = None` and `skip = skip or []` is a quick path forward, of course).

Reply:
It's something I should remember by now.  Good catch.